src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -292,221 +260,11 @@ else if (vehicleObj.getDouble("passengerLoad") <= 95)
-						tripUpdate.setStopTimeUpdate(h-1, newStopTimeUpdate);
-						preTime = timeStamp;
-//						System.out.println("** h = "+ h+ " is "+ timeStamp+ " routeID: "+tripUpdate.getTrip().getRouteId()+ " vehicle: "+tripUpdate.getVehicle().getId());
-//						System.out.println("\t DELAY:"+ delay + ", stop time of h-1 change to "+ tripUpdate.getStopTimeUpdate(h-1).getArrival().getTime()+ " from "+ preTime);
-
-					} else {
-//						System.out.println(" h = " + h+" routeID: "+tripUpdate.getTrip().getRouteId()+ " vehicle: "+tripUpdate.getVehicle().getId());
-
-						List <StopTimeUpdate> allStopUpdates = tripUpdate.getStopTimeUpdateList();
-						tripUpdate.clearStopTimeUpdate();
-						TripUpdate.Builder newTripUpdate = tripUpdate.clone();
-
-						// we have to send out the old tripUpdate, but before that the rest of stopTimes should be deleted from it
-						newTripUpdate.addAllStopTimeUpdate(allStopUpdates.subList(0, h));
-						entity ++;
-						tripUpdateEntity.setId(Integer.toString(entity));
-						tripUpdateEntity.setTripUpdate(newTripUpdate);
-						tripUpdates.addEntity(tripUpdateEntity.build());
-						//System.out.println("what has been sent: size = "+ newTripUpdate.getStopTimeUpdateList().size()+ " stoptime");
-
-						tripUpdate.addAllStopTimeUpdate(allStopUpdates.subList(h, noStopTimes));
-						//System.out.println("current tripUpdate size = "+ tripUpdate.getStopTimeUpdateList().size()+ " stoptime");
-						preTime = 0;
-						noStopTimes = noStopTimes - h;
-						h = -1;
-						StartTimes startTimes;
-						if (routeVehicleStartTimeMap.containsKeys(route, vehicleId))
-							startTimes = routeVehicleStartTimeMap.get(route, vehicleId);
-						else{
-							startTimes = new StartTimes(startTime, "0");
-							routeVehicleStartTimeMap.put(route, vehicleId, startTimes);
-						}
-
-						String previousStartT = startTimes.previousStartT;
-						TripDescriptor.Builder newTripDescriptor = TripDescriptor.newBuilder();
-						newTripDescriptor.setTripId(trip);
-						newTripDescriptor.setRouteId(route);
-						newTripDescriptor.setStartTime(previousStartT);
-						newTripDescriptor.setScheduleRelationship(ScheduleRelationship.UNSCHEDULED);
-						tripUpdate.setTrip(newTripDescriptor);
-						//System.out.println("second startTime = "+ tripUpdate.getTrip().getStartTime()+ "  &&&& first startTime = "+ newTripUpdate.getTrip().getStartTime());
-					}
-				}
-				else
-					preTime = timeStamp;
-			}
-
-			entity ++;
-
-			tripUpdateEntity.setId(Integer.toString(entity));
-			tripUpdateEntity.setTripUpdate(tripUpdate);
-			tripUpdates.addEntity(tripUpdateEntity.build());
-		 }
-		 */
-        // ----- END TRIP UPDATES ---------------
-        _tripUpdatesSink.handleFullUpdate(tripUpdates);
-        _log.info("stoIDs extracted: " + tripUpdates.getEntities().size());
-        // System.out.println("stoIDs extracted: " + tripUpdates.getEntityCount());
-
+        // ---- TRIP UPDATES REMOVED --------------------

```

Comment:
Any remaining comments here that don't reference existing code should at least reference an issue where we can find more information about, in this case, why trip updates were removed.  So let's change this to:

`// TripUpdates feeds removed due to instability converting from Syncromatics API to frequency-based feed (see #24)

And we should add a short description to the initial PR description in #24 as to why we're removing TripUpdates.